src/views/PhoneSignIn.vue

```diff
@@ -53,8 +53,12 @@ export default {
           phone: this.phone_input,
         })
         .then(() => {
-          if (this.$route.params.id) {
-            this.$router.push( {path: "/play/" + this.$route.params.id})
+          if (this.$route.params.id){
+            if (this.$route.params.type && this.$route.params.type == 'experiment') {
+              this.$router.push({path: "/experiment/" + this.$route.params.id})

```

Comment:
`type` is not used here? Shouldn't it be part of the URL? (I'm probably understanding the logic wrong!)

Reply:
The `type` here is just for the phone sign in component to know what type of component invoked it. Here, the `type == 'experiment'` check ensures that if the sign in component was invoked from the ABTesting component, it should redirect back to it. Does that make sense?

Reply:
Ah understood yes yes cool cool